### PR TITLE
Updated Uniswap v2 to schema version 1.1.0

### PR DESCRIPTION
**Pull request by steegecs**

### CODE REVIEW

**Comment by this-username-is-taken** [comment]
you need to resolve merge conflicts in the code

File: subgraphs/_reference_/src/common/rewards.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
+//////////////////////////
+// AUTHOR: CHRIS STEEGE //
```
Comment (this-username-is-taken) [inline subgraphs/_reference_/src/common/rewards.ts:2]: Can you remove this? People can find the author with `git blame`. Also version is not needed for a helper lib

**Comment by this-username-is-taken** [comment]
You should remove all `.DS_Store`s in the code. I've added it to .gitignore in latest commit in master so this will automatically be ignored going forward

File: subgraphs/_reference_/src/common/rewards.ts (hunk 1 above)
Comment (steegecs) [reply subgraphs/_reference_/src/common/rewards.ts]: Yeah - I removed this file from this pull request. I'll change it in the other.

**Comment by steegecs** [comment]
> you need to resolve merge conflicts in the code

All the merge conflict are resolved now.

**Comment by steegecs** [comment]
> You should remove all `.DS_Store`s in the code. I've added it to .gitignore in latest commit in master so this will automatically be ignored going forward

Removed

**Pull request merged**